test/public.spec.js

```diff
@@ -144,6 +144,38 @@ describe('imageChecker', () => {
         expect(imageOverlays.length).toEqual(2);
       });
 
+      it('should refresh images if the window scrollX changes', () => {
+        let intervalFn = window.setInterval.calls.mostRecent().args[0];
+        createDomNodes([
+          createImg()
+        ]);
+        window.scrollX += 1;
+
+        intervalFn();
+
+        let imageOverlays = document.querySelectorAll('.ncc-image-checker-overlay');
+        expect(imageOverlays.length).toEqual(2);
```

Comment:
@tabanliviu COVERAGE - It's not clear why the assertion is related to the describe block.